assets/js/rootLantiq.js

```diff
@@ -1,20 +1,28 @@
-async function waitUbootStop(writer, reader, sfpModel, outputMsgCallback) {
-    const interval = setInterval(function() {
-        writer.write(String.fromCharCode(3));
-    }, 10);
+const LOAD_ADDR = "80800000"
+const IMAGE0_ADDR = "C0000 740000";
```

Comment:
are these values the same for all sticks or only for lantiq MA/SF/FS?

Reply:
Only for Lantiq/MA/SF/FS (rootLantiq.js only supports these)